lib/stripe_mock/instance.rb

```diff
@@ -87,6 +88,7 @@ def initialize
       @country_spec = {}
       @tax_rates = {}
       @checkout_sessions = {}
+      @topups = Data.mock_disputes(['tu_05RsQX2eZvKYlo2C0FRTGSSA','tu_15RsQX2eZvKYlo2C0ERTYUIA', 'tu_25RsQX2eZvKYlo2C0ZXCVBNM', 'tu_35RsQX2eZvKYlo2C0QAZXSWE', 'tu_45RsQX2eZvKYlo2C0EDCVFRT', 'tu_55RsQX2eZvKYlo2C0OIKLJUY', 'tu_65RsQX2eZvKYlo2C0ASDFGHJ', 'tu_75RsQX2eZvKYlo2C0EDCXSWQ', 'tu_85RsQX2eZvKYlo2C0UJMCDET', 'tu_95RsQX2eZvKYlo2C0EDFRYUI'])
```

Comment:
mock_disputes?

wouldnt you just want it to be `{}`

Reply:
@eyefodder Had a typo in a pattern I was following from the codebase.  Resolved now.